diagnostics-app/src/app/core/enums/global.enums.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
regarding file name: do you plan to only define enums here? or can include variables as well? if the latter, then we can rename it global.vars.ts. Up to you.

Reply:
Yeah mostly we will have two folders `enums` and `interfaces`. I'll add the `interfaces` folder when we need them.